macros/adapters/bigquery/pageviews/snowplow_page_views.sql

```diff
@@ -183,8 +183,8 @@ page_pings as (
 
   select
     page_view_id,
-    min(timestamp(collector_tstamp)) as page_view_start,
```

Comment:
BigQuery raises an error on `timestamp(<timestamp_column>)`, but what about `cast(<timestamp_column> as timestamp)`?

Reply:
It worked, tysm!